modules/install-sno-ibm-power.adoc

```diff
@@ -0,0 +1,194 @@
+----
++
+[source,terminal]
+----
+$ wget ${RHCOS_URL}/rhcos-live-initramfs.ppc64le.img -o initramfs.img
+----
++
+[source,terminal]
+----
+$ cd /var//var/www/html/install/
+----
++
+[source,terminal]
+----
+$ wget ${RHCOS_URL}/rhcos-live-rootfs.ppc64le.img -o rootfs.img
+----
+
+. To create the ignition file for SNO, you need to create the `install-config.yaml` file:
+
+.. Use the following commands to create the work directory to hold the file:
++
+[source,terminal]
+----
+mkdir -p ~/sno-work
+cd ~/sno-work
+----
+
+.. Use the following sample file can to create the required `install-config.yaml` in the `~/sno-work` directory:
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: <domain> <1>
+compute:
+- name: worker
+  replicas: 0 <2>
+controlPlane:
+  name: master
+  replicas: 1 <3>
+metadata:
+  name: <name> <4>
+networking: <5>
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16 <6>
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  none: {}
+bootstrapInPlace:
+  installationDisk: /dev/disk/by-id/<disk_id> <7>
+pullSecret: '<pull_secret>' <8>
+sshKey: |
+  <ssh_key> <9>
+----
+<1> Add the cluster domain name.
+<2> Set the `compute` replicas to `0`. This makes the control plane node schedulable.
+<3> Set the `controlPlane` replicas to `1`. In conjunction with the previous `compute` setting, this setting ensures the cluster runs on a single node.
+<4> Set the `metadata` name to the cluster name.
+<5> Set the `networking` details. OVN-Kubernetes is the only allowed network plugin type for single-node clusters.
+<6> Set the `cidr` value to match the subnet of the {sno} cluster.
+<7> Set the path to the installation disk drive, for example, `/dev/disk/by-id/wwn-0x64cd98f04fde100024684cf3034da5c2`.
+<8> Copy the {cluster-manager-url-pull} and add the contents to this configuration setting.
+<9> Add the public SSH key from the administration host so that you can log in to the cluster after installation.
+
+.. Use the following commands to download the `openshift-install` image to create the ignition file and copy it to the http directory:
++
+[source,terminal]
+----
+$ wget https://mirror.openshift.com/pub/openshift-v4/ppc64le/clients/ocp/4.12.0/openshift-install-linux-4.12.0.tar.gz
+----
++
+[source,terminal]
+----
+$ tar xzvf openshift-install-linux-4.12.0.tar.gz
+----
++
+[source,terminal]
+----
+$ ./openshift-install --dir=~/sno-work create create single-node-ignition-config
+----
++
+[source,terminal]
+----
+$ cp ~/sno-work/single-node-ignition-config.ign /var/www/html/ignition/sno.ign
+----
++
+[source,terminal]
+----
+$ restorecon -vR /var/www/html || true
+----
++
+Bastion now has all the required files and is properly configured in order to install {sno}.
+
+.Procedure
+
+There are two steps for SNO installation. First the {sno} logical partition (LPAR) needs to boot up with PXE, then you need to monitor the installation progress.
```

Comment:
I don't think that SNO is an accepted abbreviation: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#single-node-openshift

Reply:
okay. Replaced all the instances in the PR.